hips/draw/simple.py

```diff
@@ -117,6 +117,30 @@ def _fetch_tiles(self) -> HipsTile:
             tile = HipsTile.fetch(tile_meta, url)
             yield tile
 
+    @staticmethod
+    def _measure_tile_shape(corners: tuple) -> List[list]:
+        """Compute tile edges and diagonals."""
+        x, y = corners
+
+        def compute_distance(i: int, j: int) -> float:
+            """Compute distance between two points."""
+            return np.sqrt((x[i] - x[j]) ** 2 + (y[i] - y[j]) ** 2)
+
+        edges = [compute_distance((i + 1) % 3, i) for i in range(4)]
+        diagonals = [compute_distance(0, 2), compute_distance(1, 3)]
+        ratio = float(np.min(diagonals) / np.max(diagonals))
+
+        return [edges, diagonals, ratio]
+
+    @staticmethod
+    def _is_tile_distorted(self, corners: tuple) -> bool:
```

Comment:
Remove `self` argument here, it's never passed to `@staticmethod`s.
You then have to call `SimpleTilePainter._measure_tile_shape(corners)` instead of `self._measure_tile_shape(corners)`.
Overall my suggestion here would be to move these two functions just out of the `SimpleTilePainter` class, and make them standalone functions and put them above or below the `SimpleTilePainter` class.